vowpalwabbit/parse_regressor.cc

```diff
@@ -18,6 +18,8 @@ using namespace std;
 #include <math.h>
 #include <algorithm>
 #include <stdarg.h>
+#include <numeric>
```

Comment:
Does this add new C++ prerequisite libraries to vw?  If yes we will have to update build docs as well.